app/services/abandon_solution_mentorship.rb

```diff
@@ -0,0 +1,11 @@
+class AbandonSolutionMentorship
+  include Mandate
+
+  initialize_with :solution_mentorship
+
+  def call
+    solution_mentorship.update(abandoned: true)
```

Comment:
If this fails, we need some way for us to know what the error is. We can either use `#update!` or send errors through Bugsnag.

Reply:
Yeah. Good point. I've not edited this in this PR, but I'll change it :)